src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form  
+      data-qa="big"
+      class="form-margin"
```

Comment:
not good class name 
you can name it just search-form and then add margin in css